pkg/controller/util/status.go

```diff
@@ -157,6 +161,47 @@ func MapCephClusterNegativeConditions(conditions *[]conditionsv1.Condition, foun
 	}
 }
 
+
+// MapExternalCephClusterNegativeConditions maps the status states from CephCluster resource into ocs status conditions.
+// This will only look for negative conditions: !Available, Degraded, Progressing
+func MapExternalCephClusterNegativeConditions(conditions *[]conditionsv1.Condition, found *cephv1.CephCluster) {
```

Comment:
Use variable for all the `Reason`s and `Message`s.

Reply:
Either use `ExternalCephCluster` or `External CephCluster`, not both.

Reply:
> Either use ExternalCephCluster or External CephCluster, not both.

Are you talking about the argument we are passing to this function or the function name ?

Reply:
I am talking about the `Message`s.

Reply:
Made everything External CephCluster !
As for using variable in Reason and Message, We decided to use the existing MapCephClusterNegativeConditions and modified it to suit our needs. In that function, there were no variables for reason and message. Let me know if this fix is required ?

Reply:
I'd prefer using variables at least for `Reason`s like we do with `Phase`.

Reply:
Added variables for reason